subsys/bluetooth/host/classic/shell/avrcp.c

```diff
@@ -0,0 +1,155 @@
+/** @file
+ *  @brief Audio Video Remote Control Profile shell functions.
+ */
+
+/*
+ * Copyright (c) 2024 Xiaomi InC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <zephyr/types.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <string.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/kernel.h>
+
+#include <zephyr/settings/settings.h>
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/classic/avrcp.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/hci.h>
+#include <zephyr/bluetooth/l2cap.h>
+
+#include <zephyr/shell/shell.h>
+
+#include "host/shell/bt.h"
+
+struct bt_avrcp *default_avrcp;
+static uint8_t avrcp_registered;
```

Comment:
Why is this not a `bool`?

Reply:
Copied the style from a2dp.c :)
Anyway, it can be a bool.

Reply:
Done.